src/python_framework_v02/troute/nhd_io.py

```diff
@@ -42,7 +42,7 @@ def read_mask(path, layer_string=None):
 def read_custom_input(custom_input_file):
     if custom_input_file[-4:] == "yaml":
         with open(custom_input_file) as custom_file:
-            data = yaml.safe_load(custom_file)
+            data = yaml.load(custom_file, Loader=yaml.SafeLoader)
```

Comment:
Make sure to rebase to the latest master -- I think we have this in there already.

Reply:
I have a fresh install. The original safe_load is in already, but after reviewing it online from a link it was displaying, the second choice is the more proper use.